docs/src/content/docs/tr/manual-setup.mdx

```diff
@@ -0,0 +1,127 @@
+---
+title: Elle Kurulum
+description: Var olan Astro projesine Starlight'ı eklemek için nasıl yapılandırıldığını öğrenin.
+---
+
+import { Tabs, TabItem } from '@astrojs/starlight/components';
+
+Yeni bir Starlight sitesi yapmanın en hızlı yolu, [Başlangıç Rehberi](/getting-started/#creating-a-new-project)'nde gösterildiği gibi `create astro` komutunu kullanmak.
```

Comment:
This link also needs the `#` part updating to match the heading translated to Turkish:

```suggestion
Yeni bir Starlight sitesi yapmanın en hızlı yolu, [Başlangıç Rehberi](/tr/getting-started/#yeni-proje-oluşturma)'nde gösterildiği gibi `create astro` komutunu kullanmak.
```